lib/booktype/urls.py

```diff
@@ -30,6 +30,9 @@
 )
 
 urlpatterns = [
+    # django log tailer
+    url(r'^logs/', include('logtailer.urls')),
```

Comment:
Hi @aerkalov ,
Urls `logtailer.urls` are using only from django admin part. Views which are under this urls are "staff only", direct access is protected, so yes, having `^_logs/` as prefix fits better.
Thanks